console-app-verify/Program.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using Telnyx.net.Services.VerifyAPI;
+
+namespace Verify
+{
+    class Program
+    {
+        private static string TELNYX_API_KEY = DotNetEnv.Env.GetString("TELNYX_API_KEY", "VarNotFound");
+        private static string TELNYX_VERIFY_PROFILE_ID = DotNetEnv.Env.GetString("TELNYX_VERIFY_PROFILE_ID", "VarNotFound");
+        static void Main(string[] args)
+        {
+            // DotNetEnv load and paste some of our variables, check if fetching correctly
+            DotNetEnv.Env.Load();
+            if (TELNYX_API_KEY == "VarNotFound")
+            {
+                Console.WriteLine("Variable not found, check your .env");
+            }
+            if (TELNYX_VERIFY_PROFILE_ID == "VarNotFound")
+            {
+                Console.WriteLine("Variable not found, check you .env file!");
+            }
```

Comment:
Combine this to a single conditional and [exit the app](https://dotnetcodr.com/2015/10/02/how-to-terminate-a-net-console-application-with-an-exit-code/) if they're not there:

```csharp
Environment.Exit(-1);
```